runtime/stackstorm-1ppc/README.md

```diff
@@ -12,70 +12,54 @@ StackStorm Docker image that runs one st2 service per container.
 
 ## Getting started
```

Comment:
Not really part of this PR but it would be nice if this directory was named `runtime/compose-1ppc/` or similar, to follow the convention set by `runtime/kubernetes-1ppc/`

Reply:
Yeah, that's a great idea. Thanks.